rotkehlchen/chain/substrate/manager.py

```diff
@@ -410,7 +410,7 @@ def _get_node_interface(self, endpoint: str) -> SubstrateInterface:
         except requests.exceptions.RequestException as e:
             message = (
                 f'{self.chain} could not connect to node at endpoint: {endpoint}. '
-                f'Connection error: {str(e)}.',
```

Comment:
@vnavascues please check this commit. Most of the times mypy catches this error, but for some reason it missed it this time.

Remember that adding a comma at the end of a multiline string in python turns it into a tuple. And that means that it will fail right at the next line when trying to log an error with the tuple.